File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.java
import java.io.*;

public class Main {
  public static void main(String[] args) throws IOException {
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    int N = Integer.parseInt(br.readLine());
    int tmp = N;
    int cnt = 0;

    while (true) {
      int a = tmp / 10;
      int b = tmp % 10;
      int c = (a + b) % 10;
      tmp = (b * 10) + c;
      cnt++;
      if (tmp == N) {
        System.out.println(cnt);
        break;
      }
    }
  }
}

File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.py
import sys
input = sys.stdin.readline

N = int(input())
tmp = N
cnt = 0

while True:
    a = tmp // 10
    b = tmp % 10
    c = (a + b) % 10
    tmp = (b * 10) + c
    cnt += 1
    if tmp == N:
        break
print(cnt)